prinz/src/test/scala/pl/touk/nussknacker/prinz/converter/mlflow/MlflowDataConverterTest.scala

```diff
@@ -2,48 +2,49 @@ package pl.touk.nussknacker.prinz.converter.mlflow
 
 import pl.touk.nussknacker.prinz.UnitTest
 import pl.touk.nussknacker.prinz.util.collection.immutable.SetMultimap
+import pl.touk.nussknacker.prinz.util.converter.mlflow.MlflowDataConverter
 
 class MlflowDataConverterTest extends UnitTest {
 
-  "MlflowDataConverter" should "convert empty data input" in {
+  val converter: MlflowDataConverter = MlflowDataConverter()
+
+  implicit class EitherOps[A](private val obj: A) {
 
+    def asLeft[B]: Either[A, B] = Left(obj)
+    def asRight[B]: Either[B, A] = Right(obj)
   }
 
-  it should "convert data in map format" in {
-    val inputData = """[
-                      |    {"a": 1,"b": 2,"c": 3},
-                      |    {"a": 4,"b": 5,"c": 6}
-                      |]""".stripMargin
+  "MlflowDataConverter" should "convert empty data input" in {
+    val inputData = """{
+                      |    "columns": [],
+                      |    "data": []
+                      |}""".stripMargin
 
-    val expected = SetMultimap[String, Int](
-      "a" -> 1, "a" -> 4,
-      "b" -> 2, "b" -> 5,
-      "c" -> 3, "c" -> 6,
-    )
+    val expected = SetMultimap.empty[String, Either[BigDecimal, String]]
 
-    // TODO call converted method and check if expected data is returned
+    converter.toMultimap(inputData) should equal (expected)
   }
 
-  it should "convert data in list format" in {
+  it should "convert data in split format" in {
     val inputData = """{
                       |    "columns": ["a", "b", "c"],
                       |    "data": [[1, 2, 3], [4, 5, 6]]
                       |}""".stripMargin
 
-    val expected = SetMultimap[String, Int](
-      "a" -> 1, "a" -> 4,
-      "b" -> 2, "b" -> 5,
-      "c" -> 3, "c" -> 6,
+    val expected = SetMultimap[String, Either[BigDecimal, String]](
```

Comment:
leave map in simple form "key" -> 42 and use map function to map your map. tests for bigger data (genarated in somehow specified way) should be added